apps/backend/src/lambdas/reconcile.ts

```diff
@@ -170,14 +174,13 @@ const runCashExceptions = async (
         : cashDates[dindex]
         ? cashDates[dindex]
         : event.period.from;
-      console.log(pastDueDate);
       if (pastDueDate) {
         appLogger.log(
           `Processing CASH EXCEPTIONS for: ${location.description} (${location.location_id})`,
           CashReconciliationService.name
         );
         appLogger.log(
-          `EXCEPTIONS DATE: ${pastDueDate}`,
+          `EXCEPTIONS DATE: ${cashDates[dindex - 2]}`,
```

Comment:
I could use an explanation on what `cashDates` looks like. I'm curious why we choose to log this particular value rather than pastDueDate

Reply:
Explained!